R/steps.R

```diff
@@ -694,8 +694,16 @@ parse_data <- function(data, dataset_id, metadata) {
     }
     out <- dplyr::bind_rows(out)
   } else {
+    # filter the data to only contain trait names which are found in var_in
     out <- df %>% filter(.data$trait_name %in% cfgChar$var_in)
     out[["value"]] <- out[["value"]] %>%  as.character()
+    # if any of the replicate values in cfgChar is one of the column names in data, bind those columns to df
+    if(any(cfgChar$replicates %in% colnames(data))){
```

Comment:
joins via bind_cols & `which` may be working, but I'd prefer use a left_join

Reply:
`left_join` doesnt work here because the columns that have been selected do not have any common columns with `df`